packages/taquito/src/contract/interface.ts

```diff
@@ -221,6 +223,16 @@ export interface ContractProvider extends StorageProvider {
    */
   txRollupSubmitBatch(params: TxRollupBatchParams): Promise<TxRollupBatchOperation>;
 
+  /**
+   *
+   * @description Submit drain delegate operation
```

Comment:
Submits a drain delegate operation i think is a bit more proper

Reply:
thanks, will push a new commit